tests/ui/issues/issue-111184.rs

```diff
@@ -0,0 +1,14 @@
+// run-pass
```

Comment:
Would tests/ui/issues/sanitizer/issue-111184.rs be a good place for it?

Reply:
`sanitizer/cfi`? idk. Also, I would probably give the test a more descriptive name that says what it's actually testing.

Reply:
> I see tidy is still complaining about this path.

Can you be more specific?

Reply:
`tests/ui/sanitizer/cfi/generators.rs` sounds like a good place. Although I'm not sure where you put other cfi ui tests if they exist already, they should all be together

Reply:
The reason I'm keeping this test separate (e.g., from https://github.com/rust-lang/rust/blob/master/tests/codegen/sanitizer-cfi-emit-type-metadata-id-itanium-cxx-abi.rs) is because it requires ` --edition=2021` and I didn't want to switch the other CFI tests to it just yet. (Maybe I can come up with an example that doesn't use async and doesn't require it.)

Reply:
i don't think having the edition be consistent is very important. using 2021 (or 2018) here but not for the other tests sounds reasonable

Reply:
I moved it to tests/ui/sanitize/issue-111184-generator-witness.rs alongside the currently existing tests/ui/sanitize/issue-72154-lifetime-markers.rs and following the same format used. Let me know if there is anything else I should change.